core/management/commands/seed.py

```diff
@@ -298,6 +327,29 @@ def create_event(visit, type):
     f.full_clean()
     f.save()
 
+
+def create_program_availability(output=True):
+    """create program availability"""
+    programs = Program.objects.all()
+    for _ in range(DEFAULT_NUMBER_PROGRAM_AVAILABILITIES):
```

Comment:
So I would do:
for program in programs:
    for day in ["Mon", "Tues", ...]:
        randomly hours, or randomly no hours or randomly more than one set of hours, (like a lunch break)
        create and save the object.

Reply:
briefly while i'm working on this, Marieke on slack said to include Saturday and Sunday

Reply:
This change completed https://github.com/CodeForPhilly/prevention-point/pull/216/commits/c0ce037ecd53e83aca5d1f9857803de3dd8929fc